### PR TITLE
Added one unknown license (EPL-1.0)

### PR DESCRIPTION
**Pull request by sunilk747**
Added one unknown license `EPL-1.0` to the existing license tree.

### CODE REVIEW

**Comment by sara-02** [comment]
[test]

File: src/license_analysis.py [hunk 1]
```diff
@@ -138,8 +140,8 @@ def _find_walks_within_type(self, v, lic_type, current_walk):
         :param current_walk: book keeping variable
         :return: None
         """
-        current_walk.append(v)
 
+        current_walk.append(v)
```
Comment (sara-02) [inline src/license_analysis.py]: Looks like you did not introduce any change in this line, avoid committing it.

File: src/license_analysis.py [hunk 2]
```diff
@@ -156,9 +158,9 @@ def _find_walks_within_type(self, v, lic_type, current_walk):
         else:
             for n in neighbours:
                 self._find_walks_within_type(n, lic_type, current_walk)
-
         current_walk.pop()
 
+
```
Comment (sara-02) [inline src/license_analysis.py]: same as above

File: src/license_analysis.py [hunk 3]
```diff
@@ -211,6 +213,7 @@ def _find_type_compatibility_classes(self):
                 # print(lic)
                 # print(list_compatibles)
 
+
```
Comment (sara-02) [inline src/license_analysis.py]: Here as well.

File: tests/test_license_analysis.py [hunk 4]
```diff
@@ -93,11 +89,23 @@ def test_compute_rep_license_conflict_2():
...
+    src_dir = "license_graph"
+    graph_store = LocalFileSystem(src_dir=src_dir)
+    synonyms_dir = "synonyms"
+    synonyms_store = LocalFileSystem(src_dir=synonyms_dir)
+    license_analyzer = LicenseAnalyzer(graph_store, synonyms_store)
+    list_licenses = ['epl 1.0', 'lgplv3+']
+    output = license_analyzer.compute_representative_license(list_licenses)
+    assert output['status'] == 'Conflict'
+    assert output['representative_license'] is None
+    expected_conflict_licenses = [('epl 1.0', 'lgplv3+')]
```
Comment (sara-02) [inline tests/test_license_analysis.py]: Are we sure to always receive these licenses in order?

Comment (sunilk747) [reply tests/test_license_analysis.py]: It give results in alphabetical order.

Comment (sara-02) [reply tests/test_license_analysis.py]: @sunilk747 thanks.

**Comment by sara-02** [comment]
@sunilk747 Thanks LGTM

**Pull request merged**